sources/packages/web/.eslintrc.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
   rules: {
     "no-console": process.env.NODE_ENV === "production" ? "warn" : "off",
     "no-debugger": process.env.NODE_ENV === "production" ? "warn" : "off",
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
Should we really disable it? It will allow us to incorrectly leave other untyped variables in the code because all of them will be considered "any", right?
I would recommend keeping it.